yahoo_fantasy_api/yhandler.py

```diff
@@ -323,9 +331,10 @@ def get_draftresults_raw(self, league_id):
         """
         return self.get("league/{}/draftresults".format(league_id))
 
-    def _build_player_stats_uri(self, game_code, player_ids, req_type, date,
+    def _build_player_stats_uri(self, league_id, player_ids, req_type, date,
```

Comment:
nit: you may want to rebase your branch or create a new one. This change looks like it's already committed in master. It makes for a tougher review to see this in there.